tests/rpc/test_rpc_telegram.py

```diff
@@ -698,7 +708,9 @@ def test_monthly_handle(default_conf, update, ticker, limit_buy_order, fee,
 
     trades = Trade.query.all()
     for trade in trades:
-        trade.update_trade(oobj)
+        trade.open_rate = oobj.safe_price
+        trade.amount = oobj.safe_amount_after_fee
```

Comment:
I do fear that this will lead to more problems down the road.

A proper method (using update_trade/update_order) seems to work for other tests?